### PR TITLE
Target doc updates for sessionId and tntId new APIs

### PR DESCRIPTION
**Pull request by swarna04**

### CODE REVIEW

File: using-mobile-extensions/adobe-target/target-event-reference.md [hunk 1]
```diff
@@ -85,8 +85,8 @@ This event is a request to retrieve Target identities, such as third-party IDs a
 
 This event will be generated by the API that is:
 
-* Requesting a Target Identity \(`getThirdPartyId`, `getTntId`\).
-* Setting the third-party ID \(`setThirdPartyId`\).
+* Requesting a Target Identity \(`getThirdPartyId`, `getTntId`, `getSessionId`\).
+* Setting a Target Identity \(`setThirdPartyId`, `setTntId`, `setSessionId`\).
```
Comment (calebk1) [inline using-mobile-extensions/adobe-target/target-event-reference.md:89]: Same comment here - no need for the \\.

**Review by calebk1 — COMMENTED** [review]
Just a few comments!

**Comment by swarna04** [comment]
I've updated the pr as per your feedback @calebk1

**Review by calebk1 — APPROVED** [review]
LGTM!

**Pull request merged**